client/src/document/molecules/_breadcrumb-locale-container.scss

```diff
@@ -0,0 +1,38 @@
+.breadcrumb-locale-container {
+  align-items: center;
+  background-color: $neutral-550;
```

Comment:
From experience, I can tell, that setting only one of color/background-color can cause problems, if the other is defined by a user.
E.g. light color because I preferred dark mode in my OS settings.

Reply:
True, but the `color` is set for the components used inside this wrapper.